Assets/Scenes/MainMenu.unity

```diff
@@ -947,11 +947,7 @@ RectTransform:
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0.5, y: 0.5}
   m_AnchorMax: {x: 0.5, y: 0.5}
-<<<<<<< HEAD
   m_AnchoredPosition: {x: 227, y: -144}
-=======
-  m_AnchoredPosition: {x: 217, y: -214}
```

Comment:
Please accept line 953 over head. Two other PRs accept this line. Just so we merge in the same change

Reply:
amended